test/fluree/db/policy/parsing_test.clj

```diff
@@ -111,8 +111,31 @@
                   :roles    #{sid-userRole}}
                  policy-alice)
               "Policies for only :ex/userRole should return")))
-
-
+      (testing "Root policy contains {:root? true} for each applicable :f/action"
+        (let [policy-root  (-> @(fluree/promise-wrap (policy/policy-map db root-did :ex/rootRole nil))
+                               replace-policy-fns)
+              sid-root-did @(fluree/internal-id db root-did)
```

Comment:
Sorry, I'm not very familiar with all the policy work yet, but why do we need to use raw internal subject ids as the :ident?

Reply:
This test is just checking that the internal implementation of policy works correctly. Users supply `did`s as the `:identity`, and don't need to use internal subject ids at all. Internally, though, we look up the subject id of the "active" identity and use it when calculating which restrictions apply to that identity.

Does that answer the question, or not quite?

Reply:
Ah, yes, now I get it :+1: